memq-actor/src/main/java/io/appform/memq/retry/config/CountLimitedExponentialWaitRetryConfig.java

```diff
@@ -18,36 +21,34 @@ public class CountLimitedExponentialWaitRetryConfig extends RetryConfig {
 
     @Min(1)
     @Builder.Default
-    int maxAttempts = 1;
+    private int maxAttempts = 1;
 
-    @Min(1)
-    @Builder.Default
-    int delayInMillis = 1;
-
-    @Min(2)
+    @Valid
+    @NotNull
     @Builder.Default
-    int maxDelayInMillis = 100;
+    private Duration waitTime = Duration.ofMillis(10);
 
-    @Min(1)
+    @Valid
+    @NotNull
     @Builder.Default
-    int maxTimeInMillisBetweenRetries = 1;
+    private Duration maxWaitTime = Duration.ofMillis(1_000);
 
     @DecimalMin("1.1")
     @Builder.Default
-    double multipier = 2.0D;
+    private double multipier = 2.0D;
 
     @Builder
     @Jacksonized
```

Comment:
If you are using builder and jacksonized, I think you can use `@Value` instead of `@Data`

Reply:
Done